lib/configs/react.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  env: {
+    browser: true
+  },
+  plugins: ['github', 'jsx-a11y'],
+  extends: ['plugin:jsx-a11y/recommended'],
```

Comment:
I'm assuming that any config we want to recommend as part of "react" belongs here, so I added the `jsx-a11y` plugin. does this look right?